tools/unitsync/CMakeLists.txt

```diff
@@ -7,12 +7,12 @@
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "../..")
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 
-find_package_static(ZLIB REQUIRED)
+find_package_static(ZLIB 1.2.7 REQUIRED)
```

Comment:
Why pin version here and in other places? Not only zlib but also other libraries?

Reply:
I pinned the minimum version for (I think unless I missed one) each of the dependencies that I updated. As for the choice of minimum version, I picked the minimum of the ones that the dependency scripts built (linux and window scripts were a bit inconsistent). I'm planning on eventually updating all the dependencies in future PRs, I just wanted to limit the amount to review for each one.